crates/matrix-sdk/src/room/joined.rs

```diff
@@ -170,37 +170,39 @@ impl Joined {
     /// if let Some(room) = client.get_joined_room(&room_id) {
     ///     room.typing_notice(true).await?
     /// }
-    /// # matrix_sdk::Result::Ok(()) });
+    /// # Result::<_, matrix_sdk::Error>::Ok(()) });
```

Comment:
Why is this better?

Reply:
It's not, apparently the default isn't infered in this case, so this change is necessary with the `Result` definition update.